lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -108,16 +109,22 @@ class DataDownload
           @$download_display_text.html data['grading_config_summary']
 
     @$calculate_grades_csv_btn.click (e) =>
+      @onClickGradeDownload @$calculate_grades_csv_btn, "Error generating grades. Please try again."
+
+    @$problem_grade_report_csv_btn.click (e) =>
+      @onClickGradeDownload @$problem_grade_report_csv_btn, "Error generating weighted problem report. Please try again."
```

Comment:
Is this a user-visible error msg? Pls fix the report name "weighted problem report -> "problem grade report."